modules/cucumber/modules/core/src/main/java/com/smartbear/readyapi/readyapi4j/cucumber/CucumberRecipeBuilder.java

```diff
@@ -0,0 +1,104 @@
+package com.smartbear.readyapi.readyapi4j.cucumber;
+
+import com.google.common.collect.Lists;
+import com.smartbear.readyapi.client.model.TestCase;
+import com.smartbear.readyapi.client.model.TestStep;
+import com.smartbear.readyapi4j.execution.Execution;
+import com.smartbear.readyapi4j.support.AssertionUtils;
+import cucumber.api.Scenario;
+import cucumber.api.java.After;
+import cucumber.runtime.java.guice.ScenarioScoped;
+
+import javax.inject.Inject;
+import java.util.List;
+
+/**
+ * Class that builds and runs the recipe to be executed
```

Comment:
this class "merges" readyapi4j into the cucumber framework via the @After and @ScenarioScoped annotations - perhaps it should be named accordingly/differently?